src/shared/table/HeaderCell.js

```diff
@@ -15,27 +15,27 @@ const HeaderCell = (props) => {
   return name === 'Categories' ? (
     <th className="table__header-cell">Categories</th>
   ) : (
-      <th className="table__header-cell">
-        <div className="header-cell__title" onClick={handleClick}>
-          <span className="header-cell__framework">{name}</span>
-          <span className="header-cell__icon">
-            <Icon handleClick={null} icon={'info'} />
-          </span>
-        </div>
-        <div className="header-cell__org">{organization}</div>
-        <div className="header-cell__doc">
-          <a
-            className="header-cell__doc-link"
-            href={url}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            <Icon onClick={null} icon={'external_link'} />
-            Original Document
+    <th className="table__header-cell">
+      <div className="header-cell__title" onClick={handleClick}>
```

Comment:
This should be a `<button>`. You can also remove the `<span>` tags inside of it.